SugarCRM sources 1) Accounts, Contacts 2) Events documentation

mohan-h opened the pull request:
Initial commit

There are 2 SugarCRM sources in this documentation PR commit
1) Accounts, Contacts 
2) Events

mohan-h committed:
SugarCRM sources 1) Accounts, Contacts 2) Events documentation

Initial commit

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,965 @@
+
+To retrieve relationship data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the below [!DNL SugarCRM] APIs :
+   1. [Account API](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) to retrieve your companies, and the 
+   1. [Contact API](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) to retrieve the people. 
```
We can shorten this to: The SugarCRM source leverages the [Accounts](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) and [Contacts](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) endpoints of the SugarMarket RESTFUL API to retrieve relationship data setup from SugarCRM.

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,965 @@
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Can we add a link to where a user can start creating an account? Suggesting some slight rewording here:

"The first step in creating a SugarCRM Accounts & Contacts source connection is to ensure that you have the following:

* A SugarMarket account. SugarMarket is SugarCRM’s marketing automation platform. To create an account, visit [LINK HERE].
* A unique API username and account separate from any user account associated with the marketing or sales process. This unique username and account combination must have API access permissions. For more information on the process to set up an account, visit the SugarMarket RESTFUL API documentation."

mohan-h replied:
* It appears the account can only be obtained through an Account manager. Have mentioned the same.
* Thanks the rewording, reads better.

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,965 @@
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| *`Host`* | API endpoint. | `developer.salesfusion.com` |
```
Can we make the description for Host into full sentence, so: "The API endpoint that the source connects to." or something like that.

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,965 @@
+
+## Connect [!DNL SugarCRM Accounts & Contacts] to Platform using the [!DNL Flow Service] API
+
+The following tutorial walks you through the steps to create a [!DNL SugarCRM Accounts & Contacts] source connection and create a dataflow to bring [!DNL SugarCRM Accounts & Contacts] data to Platform using the [[!DNL Flow Service] API](https://www.adobe.io/experience-platform-apis/references/flow-service/).
```
we can shorten the middle of this to: "...to create a [!DNL SugarCRM Accounts & Contacts] source connection and dataflow to bring..."

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,965 @@
+The following request creates a source connection for [!DNL SugarCRM Accounts & Contacts]:
+
+If you need to work with the [!DNL SugarCRM] Accounts API the `object_type` property value should be `accounts`.
+
```
We can circumvent the two examples by using tabs. Essentially, right before the shell example, we can add:

`>[!BEGIN TABS]`

`>[!TAB Accounts]`

*request example here*

`>[!TAB Contacts]`

*request example here*

`>[!ENDTABS]`

Then we can just repeat the same for response. Doc here for additional guidance: https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html?lang=en#tabs

carlojavier replied:
We can apply the same to the explore examples as well

mohan-h replied:
Introduced on both the UI and API pages for the SugarCRM Accounts & Contacts connector.

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,109 @@
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
We can remove this - I will create a separate page to contain the pre-reqs for SugarCRM

mohan-h replied:
Removed from UI
  ```help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md```
Have only preserved it within the API
  ```help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md```

carlojavier commented on help/sources/tutorials/api/create/crm/sugarcrm-events.md:
```diff
@@ -0,0 +1,662 @@
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Let's remove this. I will make a separate page on experience league to contain the pre-reqs for SugarCRM. keep the one for accounts though

mohan-h replied:
Removed from the API
  ```help/sources/tutorials/api/create/crm/sugarcrm-events.md```

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-events.md:
```diff
@@ -0,0 +1,91 @@
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Same as above, we can remove this

mohan-h replied:
Removed from UI
  ```help/sources/tutorials/ui/create/crm/sugarcrm-events.md```

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-events.md:
```diff
@@ -0,0 +1,91 @@
+![Platform UI screenshot showing an example schema for SugarCRM Events](../../../../images/tutorials/create/sugarcrm-events/sugarcrm-schema-events.png?lang=en)
+
+>[!WARNING]
+> When mapping the schema ensure you also map the mandatory event_id and timestamp fields required by Platform.
```
Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:
>[!WARNING]
>
>When mapping...

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,109 @@
+   ![Screenshot from the SugarMarket Contacts page displaying list of contacts](../../../../images/tutorials/create/sugarcrm-accounts-contacts/sugarmarket-contacts.png?lang=en)
+
+>[!NOTE] 
+> The [!DNL SugarMarket] pages do not include the deleted object counts. However, data retrieved through this source will also include the deleted count, these would be marked with a deleted flag.
```
Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:

>[!NOTE]
>
>The [!DNL SugarMarket]...

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md:
```diff
@@ -0,0 +1,109 @@
+
+To validate that you have correctly set up the source and [!DNL SugarCRM Accounts & Contacts] data is being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Accounts & Contacts] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
```
Need **[!UICONTROL]** around View Dataflows

carlojavier commented on help/sources/tutorials/ui/create/crm/sugarcrm-events.md:
```diff
@@ -0,0 +1,91 @@
+
+To validate that you have correctly set up the source and [!DNL SugarCRM Events] data is being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Events] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
```
Need **[!UICONTROL]** around View Dataflows

carlojavier reviewed (changes requested):
Hi @mohan-h review completed. Looking good so far! Pls let me know if you have any questions

mohan-h committed:
Post review updates

Updates against review https://github.com/AdobeDocs/experience-platform.en/pull/148#pullrequestreview-1198510334
